packages/edit-post/src/components/header/index.js

```diff
@@ -44,16 +49,21 @@ function Header( { setEntitiesSavedStatesCallback } ) {
 
 	const isLargeViewport = useViewportMatch( 'large' );
 
+	const classes = classnames( {
+		'edit-post-header': true,
+		'has-reduced-ui': hasReducedUI,
+	} );
+
```

Comment:
minor comment: should we define the first CSS class just a string?

```suggestion
	const classes = classnames( 'edit-post-header', {
		'has-reduced-ui': hasReducedUI,
	} );
```

No tested, just in case 😅

Reply:
Awesome!